pandas/tests/plotting/test_boxplot_method.py

```diff
@@ -160,6 +160,13 @@ def test_boxplot_empty_column(self):
         df.loc[:, 0] = np.nan
         _check_plot_works(df.boxplot, return_type='axes')
 
+    @slow
+    def test_figsize(self):
+        df = DataFrame(np.random.rand(10, 5), columns=['A', 'B', 'C', 'D', 'E'])
+        result = df.boxplot(return_type='axes', figsize=(12,8))
```

Comment:
You'll need a comma between the 12 and 8, otherwise our style cheker will complain.